testing/adios2/engine/staging-common/run_test.in

```diff
@@ -245,9 +245,9 @@ else:
     if args.reader is None:
         args.reader = 'TestCommonClient'
 
-writer_exec_cmd = '@MPIEXEC_EXECUTABLE@ @MPIEXEC_NUMPROC_FLAG@ ' + \
+writer_exec_cmd = '@MPIEXEC_EXECUTABLE@ --oversubscribe --bind-to core @MPIEXEC_NUMPROC_FLAG@ ' + \
     str(args.num_writers)
-reader_exec_cmd = '@MPIEXEC_EXECUTABLE@ @MPIEXEC_NUMPROC_FLAG@ ' + \
+reader_exec_cmd = '@MPIEXEC_EXECUTABLE@ --oversubscribe --bind-to core @MPIEXEC_NUMPROC_FLAG@ ' + \
     str(args.num_readers)
```

Comment:
I guess I'm not sure it's the best solution to hardcode this additional flags here, since other MPIs may not understand them.

I hit the oversubscribe issue before -- one way to work around it is
```
export OMPI_MCA_rmaps_base_oversubscribe=1
```
before running the tests, which I suppose could be done in the CI config as well.